azure-batch-module/modules/batch-account/main.tf

```diff
@@ -0,0 +1,97 @@
+# Create an Azure Blob Storage linked to Azure Batch
+resource "azurerm_storage_account" "BatchStorageAccount" {
```

Comment:
- I think there's a convention to use `snake_case` for resource names [1].
- Another naming convention is to not repeat the resource type in the name [1], but rather to give it a name related to its function.

[1] https://www.terraform-best-practices.com/naming

Reply:
Done.